tests/integration/conftest.py

```diff
@@ -27,15 +27,6 @@
 logger = logging.getLogger(__name__)
 
 
-pytest_plugins = [
```

Comment:
curious why we needed to move this to `tests/conftest.py`?

Reply:
that's because I use `VSCode` to debug tests which runs `pytest` tests discovery from `tests` folder. And `pytest` gives warning that it doesn't support `pytest_plugins` defined in non-top-level `conftest.py`.
reverted